oat-sa/extension-tao-itemqti
# feat: add RTL indicator to item lang selector

Related to: https://oat-sa.atlassian.net/browse/AUT-336

Add `dir="rtl"` to `itemBody` and `div.grid-rows` if selected language is linked to the RTL direction

How to test:
- set in `config/tao/client_lib_config_registry.conf.php` languages with RTL direction (for test can be `'en-US'`)
```
'util/locale' => array(
            'decimalSeparator' => '.',
            'thousandsSeparator' => '',
            'rtl' => array(
                'ar-arb',
                'en-US'
            )
        ),
```
- create item with rtl language
- add interactions
- save item and check in XML that `dir="rtl"` is added to `itemBody` and `div.grid-rows`
